FastLM-Backend/app.py

```
@@ -0,0 +1,1469 @@
+            qr_url = f'/static/qr_images/{filename}'
+            workspace.qr_image_url = qr_url
+            workspace.updated_at = datetime.utcnow()
+            
+            db.session.commit()
+            
+            return jsonify({
+                'message': 'QR 이미지가 업로드되었습니다.',
+                'qrImageUrl': qr_url
+            })
+        else:
+            return jsonify({'message': '지원되지 않는 파일 형식입니다.'}), 400
```

Pylint (3.3.7)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)